services/api/.env.example

```diff
@@ -38,7 +38,7 @@ SENDGRID_API_KEY=
 # Required if EMAIL_TRANSPORT=smtp
 # Can use services like https://ethereal.email/ or https://mailtrap.io/ during development
 SMTP_HOST=
-SMTP_PORT=
+SMTP_PORT=0
```

Comment:
I found that even though this is defaulted in the environment configuration, I still get console errors even though I'm using SG locally.